.github/workflows/e2e-tests-chrome.yml

```diff
@@ -10,24 +10,24 @@ jobs:
       matrix:
         include:
           - name: Default
-            wp-version: 5.6
-            args: "--prod --wp-version 5.6"
+            wp-version: latest
+            args: "--prod"
           - name: ES5 Bundle
-            wp-version: 5.6
-            args: "--prod --target es5 --wp-version 5.6"
+            wp-version: latest
+            args: "--prod --target es5"
           - name: Development
-            wp-version: 5.6
-            args: "--wp-version 5.6"
+            wp-version: latest
+            args: ""
             mode: "development"
           - name: Previous WordPress Version
-            wp-version: 5.5
-            args: "--prod --wp-version 5.5"
-          - name: Relative Public Path
             wp-version: 5.6
-            args: "--prod --public-path /custom/path/ --wp-version 5.6"
+            args: "--prod --wp-version 5.6"
```

Comment:
Bumping the "Previous WordPress Version" to from 5.5 to 5.6 because Yoast 16 does not support WP 5.5.